hushline/settings/__init__.py

```diff
@@ -235,6 +236,15 @@ async def index() -> str | Response:
             flash("🫥 User not found.")
             return redirect(url_for("login"))
 
+        # Initialize the directory intro text form with existing value
+        directory_intro_text_value = (
+            OrganizationSetting.fetch_one(OrganizationSetting.DIRECTORY_INTRO).value
```

Comment:
This is making two DB calls. Assign a temp variable and then do your if/else logic.

Reply:
In #40eb92c